pkg/model/components/kubelet.go

```diff
@@ -60,6 +60,7 @@ func (b *KubeletOptionsBuilder) BuildOptions(o interface{}) error {
 	clusterSpec.Kubelet.ClusterDNS = ip.String()
 	clusterSpec.Kubelet.ClusterDomain = clusterSpec.ClusterDNSDomain
 	clusterSpec.Kubelet.NonMasqueradeCIDR = clusterSpec.NonMasqueradeCIDR
+	clusterSpec.Kubelet.anonymousAuth = fi.Bool(false)
```

Comment:
should we check that it is not set first? The user may have set it to true already.

Reply:
Good catch, adding the logic